src/features/maternal/components/Step3.tsx

```
@@ -0,0 +1,136 @@
+              <span className="font-medium text-gray-900">
+                {EDUCATION_LEVEL_LABELS[formData.education_level] ||
+                  formData.education_level}
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* Info Box */}
+      <div className="bg-blue-50 border border-blue-200 rounded-md p-3 text-sm text-blue-900">
+        ℹ️ By clicking "Submit", you agree to register for the maternal health
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)